src/frontend/komponenter/Høyremeny/Steg.ts

```diff
@@ -0,0 +1,29 @@
+export const enum Steg {
+    REGISTRERE_OPPLYSNINGER = 'REGISTRERE_OPPLYSNINGER',
+    VILKÅRSVURDERE_INNGANGSVILKÅR = 'VILKÅRSVURDERE_INNGANGSVILKÅR',
+    VILKÅRSVURDERE_STØNAD = 'VILKÅRSVURDERE_STØNAD',
+    BEREGNE_YTELSE = 'BEREGNE_YTELSE',
+    SEND_TIL_BESLUTTER = 'SEND_TIL_BESLUTTER',
+    BESLUTTE_VEDTAK = 'BESLUTTE_VEDTAK',
+    IVERKSETT_MOT_OPPDRAG = 'IVERKSETT_MOT_OPPDRAG',
+    VENTE_PÅ_STATUS_FRA_ØKONOMI = 'VENTE_PÅ_STATUS_FRA_ØKONOMI',
+    JOURNALFØR_VEDTAKSBREV = 'JOURNALFØR_VEDTAKSBREV',
+    DISTRIBUER_VEDTAKSBREV = 'DISTRIBUER_VEDTAKSBREV',
+    FERDIGSTILLE_BEHANDLING = 'FERDIGSTILLE_BEHANDLING',
+    BEHANDLING_FERDIGSTILT = 'BEHANDLING_FERDIGSTILT',
+}
+
+export const StegVerdi = new Map<Steg, string>([
```

Comment:
Vi anbefaller å bruke `Record` istællet før `Map`, fordi (tslint?) gir en warning hvis man har glømt å definiera string verdi for en enum verdi. Eks hvis du hade glømt att definere tekst-verdien for Steg.BESLUTTE_VEDTAK så hade tslint gitt dig en warning